.changeset/short-tools-worry.md

```diff
@@ -0,0 +1,6 @@
+---
+"@whereby.com/media": patch
+"@whereby.com/core": patch
```

Comment:
I think this is `minor`.

> Given a version number MAJOR.MINOR.PATCH, increment the:
> MAJOR version when you make incompatible API changes
> MINOR version when you add functionality in a backward compatible manner
> PATCH version when you make backward compatible bug fixes

source: https://semver.org

Reply:
agreed

Reply:
Ahh will fix